Terminal.Gui/Views/FrameView.cs

```diff
@@ -25,7 +25,7 @@ public FrameView ()
     private void FrameView_MouseClick (object sender, MouseEventArgs e)
     {
         // base sets focus on HotKey
-        e.Handled = InvokeCommand (Command.HotKey, ctx: new (Command.HotKey, key: null, data: this)) == true;
+        e.Handled = InvokeCommand<KeyBinding> (Command.HotKey, new ([Command.HotKey], this, this)) == true;
```

Comment:
Putting type on the new instead of the method should be identical but read better?

`InvokeCommand(Command.HotKey, new KeyBinding( ...`